doc/betweenness-centrality.adoc

```diff
@@ -1,47 +1,57 @@
-It represents the degree of which nodes stand between each other. 
-For example, in a telecommunications network, a node with higher betweenness centrality would have more control over the network, because more information will pass through that node. 
-Betweenness centrality was devised as a general measure of centrality. 
-It applies to a wide range of problems in network theory, including problems related to social networks, biology, transport and scientific cooperation.
+The original idea behind betweenness centrality was introduced by J. M. Anthonisse in his 1971 paper http://oai.cwi.nl/oai/asset/9791/9791A.pdf[The rush in a directed graph^].
+Anthonisse defined the _rush_ in a graph as the amount a node in a network has to intermediate between other nodes.
 
+Linton Freeman gave the first formal definition of betweenness centrality as one of the “three distinct intuitive conceptions of centrality” in his paper http://moreno.ss.uci.edu/23.pdf[A Set of Measures of Centrality Based on Betweenness^].
+It is calculated by a breath-first search algorithm that computes the shortest paths from every node to all other nodes.
+Ulrik Brandes' http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.11.2024&rep=rep1&type=pdf[Faster Algorithm for Betweenness Centrality^] is often used to make the calculation.
 
 == When to use it / use-cases
 
 image::{img}/betweenness_centrality.png[]
 
+_Betweenness centrality_ is useful for finding nodes that serve as a bridge from one part of a graph to another.
+As a result, it is a rudimentary measure of the control that a node exerts over the flow throughout the graph.
 
-_Betweenness centrality_ is useful in finding vertices that serve as a bridge from one part of a graph to
-another. Consequently, betweenness is a rudimentary measure of the control, that a specific node exerts
-over the flow throughout the full graph. For example, Alice in above example is the
-main connection in the graph. Were Alice to be removed, all connections in the graph would be cut off. This makes Alice “important”, because it ensures that no nodes are isolated. When using
-betweenness centrality as an analysis measure, it indicates a potential gate keeping or controlling node. 
+In the above example Alice is the main connection in the graph.
+If Alice is removed all connections in the graph would be cut off.
+This makes Alice “important” because she ensures that no nodes are isolated.
 
-It differs from the other centrality measures. A node can have quite low degree, be connected to others that have low degree, even be a long way from others on average, and still have high betweenness. Consider a node A that lies on a bridge between two groups of vertices within a network. Since any path between nodes in different groups must go through this bridge, node A acquires high betweenness even though it is not well connected (it lies at the periphery of both groups).
+The betweenness score for a node can be quite deceptive.
```

Comment:
I wouldn't call it deceptive. Maybe a different view of the importance of nodes from other centralities.

Reply:
Fair enough. How about if we say it's 'surprising' or 'not intuitive'?

Reply:
Not intuitive would be better I think